src/routes/(main)/+page.svelte

```diff
@@ -66,15 +66,19 @@
 <TalentsNavigation />
 
 <div class="randomizer-notif">
-  <a href="/randomizer" target="_blank"><img src={RandomizerIcon} alt="OW Armory icon" /></a>
+  <a href="/randomizer" target="_blank" class="icon-container">
+    <img src={RandomizerIcon} alt="OW Armory randomizer icon" height="125" />
```

Comment:
Two small things;
- The alt icon isn't terribly accessible. "Ow Armory randomizer" would be sufficient, stating that it's an icon isn't really relevant for screen readers.
- Without a width value, the layout will shift as the image loads in